config.json

```diff
@@ -0,0 +1,243 @@
+{
+  "language": "Solidity",
```

Comment:
Can we update the readme about how to generate this file as the developer setup.